Unit2_Applications/fizzbuzz.py

```diff
@@ -0,0 +1,14 @@
+print("submit any number.")
+user_input= (input())
+number = int(user_input) #variable for users input.
+multiples = 0 # using (multiples) as a start off count from 0.
+for loop in range(number):
```

Comment:
The code looks like it'll get the job done!

However, it looks like you aren't using the variable `loop` anywhere.  I'd call it `counter` or something and eliminate the `multiples` variable.  Currently `loop` and `multiples` are the same thing, so the `multiples` code is redundant!